tests/AbstractServiceTest.php

```diff
@@ -36,15 +36,18 @@ protected function setUpCache()
             ->getMock();
     }
 
-    protected function setUpMapper($mapperName, $entityBuilderMethod)
+    protected function setUpMapper($mapperName, $callback)
```

Comment:
The seccond arg is currently callable|string as we haven't retrofitted the rest of the service tests yet.